### ISSUE TITLE
[Question] Why "@preact/playwright-ct" types are not available?

### ISSUE THREAD

**Issue by chrisha22** [preactjs/playwright-ct#9]
I have a project with preact and I installed the package `preact/playwright-ct`, however, Im seeing this error when importing config on my `config.ts` file :
        "resource": "/playwright-ct.config.ts",
	"owner": "typescript",
	"code": "2307",
	"message": "Cannot find module '@preact/playwright-ct' or its corresponding type declarations."

Does anyone know how to fix it? is there a missing dependency , maybe?
@developit

**Comment by JoviDeCroock** [comment]
Hey, does it fix itself if you add

```
{
  "types": "./dist/index.d.mts"
}
```

to the package json in your node_modules, if so we are missing that field in our package. The new resolution mechanism for newer TS versions is included

**Comment by chrisha22** [comment]
@JoviDeCroock I was checking the package.json for the node_modules>@preact>playwright-ct and that field is already inside. I tried adding `"moduleResolution": "NodeNext"` in the ts.config and the error is no longer shown inside the test file import, but it is still appearing inside the `playwright-ct.config.ts` and also there is like a warning saying that NodeNext is unstable.

**Comment by JoviDeCroock** [comment]
It's not their at the first-level, only in the export-map. Mind putting it at the base level as well?

**Comment by chrisha22** [comment]
I’m sorry @JoviDeCroock I think I'm not following, do you mean adding 
{
  "types": "./dist/index.d.mts"
}

at the root level of my project's `package.json`? or where exactly?

**Comment by JoviDeCroock** [comment]
Of the package json of our package `@preact/playwright-ct` it seems to be missing from the latest release so just adding it above [this line](https://www.runpkg.com/?@preact/playwright-ct@0.1.0/package.json#13) without `nodeNext`

**Comment by chrisha22** [comment]
Like this?

 ```
 "name": "@preact/playwright-ct",
  "version": "0.0.2",
  "description": "Playwright Component Testing for Preact",
  "repository": "github:preact/playwright-ct",
  "keywords": [
    "Preact",
    "Playwright",
    "testing"
  ],
  "author": "Marvin Hagemeister <hello@marvinh.dev>",
  "license": "MIT",
  "types": "./dist/index.d.mts",
  "exports": {
    ".": {
      "types": "./dist/index.d.ts",
      "default": "./dist/index.js"
    },
    "./hooks": {
      "types": "./dist/hooks.d.mts",
      "default": "./dist/hooks.mjs"
    }
  },
```

**Comment by JoviDeCroock** [comment]
Yes, but `index.d.ts` sorry I misstyped

**Comment by chrisha22** [comment]
yes, that fixed it!, when is it gonna be available so I can update the package with that change?

**Comment by JoviDeCroock** [comment]
I mean, when someone makes a Pull Request and it gets released :sweat_smile:

**Comment by chrisha22** [comment]
@JoviDeCroock  Since you guys will add the fix for the types Issue in a next version, I'll mark this as resolved, but the problem will still present, I'll be looking forward to get this version as soon as it is available.

Thanks for helping

**Comment by chrisha22** [comment]
I tried to create a PR with the change so you guys could check it but Unfortunately I couldn’t.

**Comment by JoviDeCroock** [comment]
Published as 0.1.1
